### ISSUE TITLE
Multiple Chests on the same Map?

### ISSUE THREAD

**Issue by BlindGuyNW** [khanshoaib3/stardew-access#156]
Hi there,

Accessible Tiles/the newest Stardew Access beta only support one chest per map. They seem to pick which one is visited arbitrarily, or perhaps by proximity?

This makes placing and using multiple chests annoying because I have to keep trakc of coordinates for each one.

Any work on fixing this would be appreciated. I dont' know if it also applies to other similar cases.

**Comment by khanshoaib3** [comment]
The reason behavior is because all chests have same name so the object tracker groups all chests together.
From the next beta, we will be able to change the colors of chest and that also reflects on the name of the chest so they we will be able to differentiate from the object tracker as well. There is also a mod that adds the ability to name a chest, we'll patch that mod after or during this release.

**Comment by Flameborn** [comment]
I am guessing this is a recent change concerning the way the object tracker behaves.

It is unfortunately happening with any object which has multiple copies on the map, including crops.

Would it be possible to restore the old behavior, where if the two objects have a different coordinate, they would show up as two separate objects instead? With chests, naming them or choosing a color can solve the issue, but not all objects can be renamed unfortunately. In addition, the object closest to the player may not be the one the player wishes to navigate to in all cases.

If making the object list of categories brief is desired, this grouping behavior should be customizable I think.

**Comment by khanshoaib3** [comment]
Sure, we'll look into this.

**Comment by khanshoaib3** [comment]
> Thank you. Would it help if I raised an issue?

No need 🙂, I just mentioned ParadoxiKat (who is handling the accessible tiles merge) about it.

**Comment by khanshoaib3** [comment]
I think a way to solve this issue would to add number prefixes to objects with same names.
Also make sure that this does not affect the proximity sorting.

**Comment by Flameborn** [comment]
That's a good idea. Then, sound or speech could inform the player that there are multiple copies, so one could navigate through them. Maybe via shift+pgup and shift+pgdown.

**Comment by ParadoxiKat** [comment]
I agree that suffix is better than prefix. I like the idea of keeping them grouped but then being able to zoom in further to select which one. So for example, moving through the crops category, and it says "24 tomatoes". If you immediately navigate at that point it will just take you to the closest one.
However you can instead at this point press the keys to cycle the group, and you can focus "tomato 1" or "tomato 12" and see their individual coordinates, then navigate to "tomato 17" if that's the one you actually need.

I can work on this tomorrow now that the other big thing I was working on is stable :)

**Comment by ParadoxiKat** [comment]
b3801215cfa9667dfdc8aa99bb7d44803288c97c adds ability for static tile names to include optional number in brackets as suffix.
Example, "static_tile-common-stool[2]" will have "static_tile-common-stool" translated, then append 2. For English translation this means "Stool 2" is spoken.

Some work has also been done in 404707e5acd34ef423b0eed4eff9ef81b9c5547c to enumerate doors with the same name.
